libs/theme/src/preset.ts

```diff
@@ -50,9 +53,9 @@ const preset = {
       },
       fontSize: {
         base: font("sm", "5", "normal"),
-        small: font("sm"),
-        title: font("lg", "7", "medium"),
-        subtitle: font("sm", "5", "normal"),
+        normal: font("base", "6", "normal"),
```

Comment:
This could be categorized as a potentional major update

Reply:
Maybe not perhaps, the change now is only on line height and that could be marked in release notes.